__tests__/app.test.js

```diff
@@ -18,8 +18,23 @@ describe("GET /api", () => {
     return request(app)
       .get("/api")
       .expect(200)
-      .then(({ body: { endpoints } }) => {
-        expect(endpoints).toEqual(endpointsJson);
+      .then(({ body }) => {
+        expect(body.endpoints).toEqual(endpointsJson);
+      });
+  });
+});
+
+describe("/api/topics", () => {
+  test("GET /api/topics - responds with status 200 and an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toHaveLength(3);
+        body.topics.forEach(({ description, slug }) => {
+          expect(typeof description).toBe("string");
+          expect(typeof slug).toBe("string");
+        });
```

Comment:
Well done for asserting against the length here to ensure the test does not pass by default if we get an empty array back! :smile: